presentation/backend/src/presentation-backend/RulesetEmbedder.ts

```diff
@@ -81,19 +81,13 @@ export interface RulesetInsertOptions {
    * Callbacks that will be called before and after Element updates
    * @beta
    */
-  onElementUpdate?: UpdateCallbacks<DefinitionElement>;
+  onElementUpdate?: UpdateCallbacks<Element>;

```

Comment:
Why not use `Entity` in both insert and update cases?

Reply:
Sure, updated